www/docs/main/quickstart.md

```diff
@@ -0,0 +1,119 @@
+---
+id: quickstart
+title: Quickstart
+sidebar_label: Quickstart
+slug: /quickstart
+author: colinhacks
+author_url: https://twitter.com/colinhacks
+author_image_url: https://avatars.githubusercontent.com/u/3084745?v=4
+---
+
+## Installation
+
+**⚠️ Requirements**: tRPC requires TypeScript > 4.1 as it relies on [Template Literal Types](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html).
+
+`npm install @trpc/server`
+
+For implementing tRPC endpoints and routers. Install in your server codebase.
+
+`npm install @trpc/client`
+
+For making typesafe API calls from your client. Install in your client codebase.
+
+`npm install @trpc/react`
+
+For generating a powerful set of React hooks directly from your tRPC client. Powered by [react-query](https://react-query.tanstack.com/).
+
+`npm install @trpc/next`
+
+A set of utilies for integrating tRPC with Next.js
+
+## Defining a router
+
+Let's walk through the steps of building a typesafe API with tRPC. To start, this API will only contain two endpoints:
+
+```ts
+getUser(id: string) => { id: string; name: string; }
+createUser(data: {name:string}) => { id: string; name: string; }
+```
+
+### Create a router instance
+
+First we define a router somewhere in our server codebase:
+
+```ts
+// server/index.ts
+import * as trpc from '@trpc/server';
+export const AppRouter = trpc.router();
```

Comment:
```tsx
export const appRouter = trpc.router();
```

the instance should live under `appRouter` and the _type_ should live under `AppRouter`